core/src/components/item/item.md.scss

```diff
@@ -298,24 +203,6 @@
   @include margin($item-md-label-margin-top, $item-md-label-margin-end, $item-md-label-margin-bottom, $item-md-label-margin-start);
 }
 
-
-// Material Design Floating/Stacked Label
-// --------------------------------------------------
-
-:host(.item-label-stacked) ::slotted([slot="end"]),
-:host(.item-label-floating) ::slotted([slot="end"]) {
-  @include margin($item-md-label-slot-end-margin-top, $item-md-label-slot-end-margin-end, $item-md-label-slot-end-margin-bottom, $item-md-label-slot-end-margin-start);
-}
```

Comment:
Why were these removed if we're keeping the `ion-label` position styles?

Reply:
Good catch. I had originally removed this before I realized that I should not have. These instances were the ones I missed when reverting.